packages/gdrive-downloader/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/Akianonymus/gdrive-downloader
+TERMUX_PKG_DESCRIPTION="Download a gdrive folder or file easily, shell ftw."
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2022.03.31
+TERMUX_PKG_SRCURL=$TERMUX_PKG_HOMEPAGE.git
+TERMUX_PKG_GIT_BRANCH=master
```

Comment:
You should specify a commit hash. Without one the source tree may change for every fetch.